src/sm/sm-common.c

```diff
@@ -385,29 +383,32 @@ sm_encrypt_des_ecb3(unsigned char *key, unsigned char *data, int data_len,
 	for (ii=0; ii<data_len; ii+=8)
 		DES_ecb2_encrypt( (sm_des_cblock *)(data + ii),
 				(sm_des_cblock *)(*out + ii), &ks, &ks2, DES_ENCRYPT);
+
+	return 0;
```

Comment:
better use SC_SUCCESS here

Reply:
Thanks for review. Changed to use SC_SUCCESS also in other cases of this function.